drivers/core_demographics_report/app/models/core_demographics_report/age_calculations.rb

```diff
@@ -24,6 +24,13 @@ def age_categories
       }
     end
 
+    def age_range
```

Comment:
can we rename this `adult_and_child_age_ranges`?  There are other uses of `age_range` further down in this file within a block `age_categories.each do |age_range, age_title|` which can make the name of this method confusing.

Reply:
alternatively something like:
```
def age_range_for(category)
  return (0..17) if category == 'child'
  return (18..110) if category == 'adult'

  raise "unknown category: #{category}"
end
```

might be more clear what's going on.
I don't _think_ we're ever looping over those age ranges, so returning the range is probably better anyway.

Reply:
update this method to `age_range_for` and returning the range instead of the hash.